Scripts/RemediationScripts/Remediate-EnableEncryptionInTransitForStorageAccounts.ps1

```diff
@@ -0,0 +1,686 @@
+            $rollbackFailure = @()
+                    
+            Write-Host "Disabling secure transfer on [$(($remediatedResourceLog| Measure-Object).Count)] Storage Account(s) of subscription [$($SubscriptionId)]..." 
+           
+            $remediatedResourceLog | ForEach-Object {
+                try
+                {
+                    $output = Set-AzStorageAccount -ResourceGroupName $_.ResourceGroupName -Name $_.StorageAccountName -EnableHttpsTrafficOnly $false -ErrorAction SilentlyContinue
+                    if($output -ne $null)
+                    {
+                        $rollbackSuccess += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"
+                    }
+                    else
+                    {  
+                       Write-Host "Rollback is not successful on [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Warning)    
+                       $rollbackFailure += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"                              
+                    }
+                }             
+                catch
+                {
+                    Write-Host "Error occured while rollback on [$($_.StorageAccountName)] : [$($_.ResourceGroupName)]" -ForegroundColor $([Constants]::MessageType.Error)
+                    Write-Host  "Error: $($_)" -ForegroundColor $([Constants]::MessageType.Error)
+                    $rollbackFailure += $_ |Select-Object -Property "StorageAccountName" , "ResourceGroupName" , "ResourceId"
+                }
+            }
+
+            Write-Host $([Constants]::DoubleDashLine)
+
+            if(($rollbackSuccess|Measure-Object).Count -ne 0)
+            {   Write-Host "Rollback is successful on following  Storage Account(s):" -ForegroundColor ([Constants]::MessageType.Update)
+                $rollbackSuccess | Select-Object -Property "ResourcegroupName", "storageAccountName" ,"ResourceId" | ft   
+                write-host $([Constants]::SingleDashLine)
+
+            } 
+            
+            if(($rollbackFailure|Measure-Object).Count -ne 0)
+            {
+                Write-Host "Rollback is not successful on following Storage Account(s):" -ForegroundColor ([Constants]::MessageType.Error)
+                $rollbackFailure | Select-Object -Property "ResourcegroupName", "storageAccountName" ,"ResourceId"| ft        
+            }         
+        }
+        else 
+        {
+            Write-Host "No Storage Account(s) found in remediation log to perform rollback operation." -ForegroundColor $([Constants]::MessageType.Update)
+            Write-Host $([Constants]::DoubleDashLine)
+            break
+        }
+    }   
+    catch
+    {
+        Write-Host "Error occurred while performing rollback opeartion for remediating changes. ErrorMessage [$($_)]" -ForegroundColor $([Constants]::MessageType.error)
```

Comment:
Typo: `opeartion` to be replaced with `operation`.
`MessageType.error` to be replaced with `MessageType.Error`.